sdk/tables/azure-tables/review/tables.api.md

```diff
@@ -4,8 +4,513 @@
 
 ```ts
 
+import * as coreHttp from '@azure/core-http';
+
+// @public
+export interface AccessPolicy {
+    expiry: Date;
+    permission: string;
+    start: Date;
+}
+
+// @public
+export interface CorsRule {
+    allowedHeaders: string;
+    allowedMethods: string;
+    allowedOrigins: string;
+    exposedHeaders: string;
+    maxAgeInSeconds: number;
+}
+
+// @public
+export interface CreateEntityOptions extends coreHttp.OperationOptions {
+    queryOptions?: QueryOptions;
+    requestId?: string;
+    responsePreference?: ResponseFormat;
+    tableEntityProperties?: {
+        [propertyName: string]: any;
+    };
+    timeout?: number;
+}
+
+// @public
+export type CreateEntityResponse = TableInsertEntityHeaders & {
+    [propertyName: string]: any;
+} & {
+    _response: coreHttp.HttpResponse & {
+        bodyAsText: string;
+        parsedBody: {
+            [propertyName: string]: any;
+        };
+        parsedHeaders: TableInsertEntityHeaders;
+    };
+};
+
+// @public
+export interface CreateTableOptions extends coreHttp.OperationOptions {
+    queryOptions?: QueryOptions;
+    requestId?: string;
+    responsePreference?: ResponseFormat;
+}
+
+// @public
+export type CreateTableResponse = TableCreateHeaders & TableResponse & {
+    _response: coreHttp.HttpResponse & {
+        bodyAsText: string;
+        parsedBody: TableResponse;
+        parsedHeaders: TableCreateHeaders;
+    };
+};
+
+// @public
+export interface DeleteEntityOptions extends coreHttp.OperationOptions {
+    queryOptions?: QueryOptions;
+    requestId?: string;
+    timeout?: number;
+}
+
+// @public
+export type DeleteEntityResponse = TableDeleteEntityHeaders & {
+    _response: coreHttp.HttpResponse & {
+        parsedHeaders: TableDeleteEntityHeaders;
+    };
+};
+
+// @public
+export interface DeleteTableOptions extends coreHttp.OperationOptions {
+    requestId?: string;
+}
+
+// @public
+export type DeleteTableResponse = TableDeleteHeaders & {
+    _response: coreHttp.HttpResponse & {
+        parsedHeaders: TableDeleteHeaders;
+    };
+};
+
+// @public
+export interface Entity {
+    [propertyName: string]: any;
+    PartitionKey: string;
+    RowKey: string;
+}
+
 // @public (undocumented)
-export function helloWorld(): string;
+export interface GeoReplication {
```

Comment:
NIT: Let's remember to document this, doesn't have to happen right now though.

Reply:
The description fields for GeoReplication and Metrics is missing in the api swagger files. Ideally they should be added there by the service team, but if they don't I will add them manually.